src/Hcal/HcalTriggerGeometry.cxx

```diff
@@ -0,0 +1,114 @@
+#include "Hcal/HcalTriggerGeometry.h"
+#include <iostream>
+#include <sstream>
+#include "DetDescr/HcalGeometry.h"
+#include "Framework/ConditionsObjectProvider.h"
+#include "Framework/EventHeader.h"
+
+namespace hcal {
+
+HcalTriggerGeometry::HcalTriggerGeometry(const ldmx::HcalGeometry* hcalGeom)
+  : ConditionsObject(CONDITIONS_OBJECT_NAME),
+    hcalGeometry_{hcalGeom} {
+  
+}
+
+std::vector<ldmx::HcalDigiID> HcalTriggerGeometry::contentsOfQuad(
+    ldmx::HcalTriggerID triggerCell) const {
+  std::vector<ldmx::HcalDigiID> retval;
+  for(int iStrip=0;iStrip<4;iStrip++){
+    int strip = iStrip+4*triggerCell.superstrip();
+    if (strip >= hcalGeometry_->getNumStrips(triggerCell.section())) break;
+    retval.push_back( ldmx::HcalDigiID(triggerCell.section(), triggerCell.layer(),
+                                       strip, triggerCell.end()) );
+  }
+  return retval;
+}
+
+  std::vector<ldmx::HcalDigiID> HcalTriggerGeometry::contentsOfSTQ(
```

Comment:
Does this mean that one Super Trigger Quad (STQ) has 4 Quads (16 bars total) divided in 2 layers? i.e. for a given layer you will have only 8 bars that belong to a single STQ - and 8 STQs total?